docs-team/2015-12/december-17.md

```diff
@@ -0,0 +1,146 @@
+Attending:
+
+* David Baker
+* Lead Silber
+* Michael Kaiser-Nyman
+* Ricardo Mendes
+* Ryan Tablada
+* Todd Jordan
+
+---
+
+DB: I accidentally pushed to master.
+
+RT: Can't we make a protected branch?
+
+DB: We don't have access.
+
+MKN: Ask Leah.
+
+---
+
+MKN: Should we move troubleshooting for Windows devs into Troubleshooting.md?
+
+DB: Sounds good.
+
+---
+
+DB: Someone wants to help translate into other languages. Maybe we could do it on the major releases?
+
+MKN: We would need to figure out infrastructure. They should open an issue with a suggestion for the infrastructure.
+
+DB: Do we have Google Analytics so we can see where our visitors are?
```

Comment:
@acorncom Ping me about this on Slack next week and I can help track down the answer :)